integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -228,7 +229,10 @@ func SendRequest(t *testing.T, e deployment.Environment, state CCIPOnChainState,
 		Context: context.Background(),
 	}, []uint64{dest}, []uint64{})
 	require.NoError(t, err)
-	require.True(t, it.Next())
+
+	for it.Next() {
+		time.Sleep(5 * time.Second)
+	}
```

Comment:
What's the reason for this sleep? Adding sleeps to test code generally introduces flakiness

Reply:
removed